releaseNotes/AzureSpringCloudFeatureManagement.md

```diff
@@ -0,0 +1,22 @@
+# azure-spring-cloud-feature-management
+
+[Source code][source_code] | [Package (Maven)][package] | [Samples][samples] | [Product documentation][docs]
+
+# azure-spring-cloud-feature-management-web
+
+[Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Samples web][samples_web] | [Product documentation][docs]
+
+## 2.0.0-beta.1 - May 5, 2021
+
+* Same Feature as `com.microsoft.azure` `azure-spring-cloud-feature-management` and `com.microsoft.azure` `azure-spring-cloud-feature-management-web`
```

Comment:
Not sure what you mean here.

I think we can actually talk about renaming here because it is indeed just a renaming for the feature management libraries.